crates/typst/src/model/styles.rs

```diff
@@ -317,61 +319,73 @@ impl<'a> StyleChain<'a> {
         }
     }
 
+    /// Cast the first value for the given property in the chain,
+    /// returning a borrowed value if possible.
+    pub fn get_borrowed<'b, T: Blockable>(
+        &'b self,
+        func: ElementData,
+        id: impl Into<u8>,
```

Comment:
why generic here `impl Into<u8>` here?

Reply:
So that I can pass the enums directly, makes the generated code a bit less obscure imo

Reply:
I didn't know that enums impl Into their discriminant!

Reply:
Neither did I, I just sort of did it on accident and it worked :D
And I kept it because I think it's a neat way of doing the conversion and keeping the generated code easy to read.